zygrader/grade_puller.py

```diff
@@ -79,24 +84,30 @@ def read_canvas_csv(self):
                 f"Could not find {path}",
                 "Please download the gradebook from Canvas and put it in the place noted above",
             ]
-            self.window.create_popup("Error in Reading Master CSV", msg)
+            popup.set_message(msg)
+            self.window.run_layer(popup)
             raise GradePuller.StoppingException()
         except PermissionError:
             msg = [
                 f"Could not open {path} for reading",
                 "Please have the owner of the file grand read permissions",
```

Comment:
```suggestion
                "Please have the owner of the file grant read permissions",
```
Just found this small typo scanning through all the files.